### ISSUE TITLE
Unable to run npm run test without strange error on Windows 10

### ISSUE THREAD

**Issue by RandellDawson** [freeCodeCamp/freeCodeCamp#37610]
I get the following error when running `npm run test` with the most recent version of the `master` branch on Windows 10.  I have even tried running `npm run clean` then `npm ci` to make sure everything is up-to-date, but still get this error.  It seems to have started some time before Oct 26.  

If I switch to the curriculum folder and run `npm run test`, those tests run without a problem.

```
OPT@HP-PC MINGW64 /d/coding/fcc (master)
$ npm run test

> @freecodecamp/freecodecamp@0.0.1 test D:\coding\fcc
> npm-run-all -p test:*


> @freecodecamp/freecodecamp@0.0.1 test:client D:\coding\fcc
> cd ./client && npm test && cd ../


> @freecodecamp/freecodecamp@0.0.1 test:curriculum D:\coding\fcc
> cd ./curriculum && npm test && cd ../


> @freecodecamp/freecodecamp@0.0.1 test:guide-formatting D:\coding\fcc
> node ./tools/scripts/ci/ensure-guide-formatting.js


> @freecodecamp/freecodecamp@0.0.1 test:lint D:\coding\fcc
> echo 'Warning: TODO - Define Linting tests.'


> @freecodecamp/freecodecamp@0.0.1 test:search-indexing D:\coding\fcc
> jest ./search-indexing


> @freecodecamp/freecodecamp@0.0.1 test:server D:\coding\fcc
> cd ./api-server && npm test && cd ../

'Warning: TODO - Define Linting tests.'

> @freecodecamp/freecodecamp@0.0.1 test:tools D:\coding\fcc
> jest ./tools


> @freecodecamp/freecodecamp@0.0.1 test:utils D:\coding\fcc
> jest ./utils/*.js


> @freecodecamp/curriculum@0.0.0-next.4 pretest D:\Coding\fcc\curriculum
> cd ../client && npm run build:workers


> @freecodecamp/client@1.0.0 test D:\Coding\fcc\client
> jest


> @freecodecamp/api-server@0.0.1 test D:\Coding\fcc\api-server
> jest


> @freecodecamp/client@1.0.0 build:workers D:\Coding\fcc\client
> node --max_old_space_size=7168 node_modules/webpack-cli --env.production --config ./webpack-workers.js --progress

Hash: 67d106f126bbdd122446
Version: webpack 4.41.2
Time: 6051ms
Built at: 10/29/2019 6:32:17 AM
                               Asset       Size  Chunks                          Chunk Names
                   enzyme-adapter.js  185 bytes    2, 1  [emitted]               enzyme-adapter
               enzyme-adapter.js.map   94 bytes    2, 1  [emitted] [dev]         enzyme-adapter
                           enzyme.js  177 bytes    1, 2  [emitted]               enzyme
                       enzyme.js.map   86 bytes    1, 2  [emitted] [dev]         enzyme
                     frame-runner.js    187 KiB       3  [emitted]               frame-runner
                 frame-runner.js.map    899 KiB       3  [emitted] [dev]         frame-runner
                     sass-compile.js   1.34 KiB       4  [emitted]               sass-compile
                 sass-compile.js.map   5.87 KiB       4  [emitted] [dev]         sass-compile
                        sass.sync.js   4.27 MiB          [emitted]        [big]
                   test-evaluator.js    166 KiB    5, 6  [emitted]               test-evaluator
               test-evaluator.js.map    880 KiB    5, 6  [emitted] [dev]         test-evaluator
                     vendors~chai.js   67.9 KiB       6  [emitted]               vendors~chai
                 vendors~chai.js.map    423 KiB       6  [emitted] [dev]         vendors~chai
           vendors~enzyme-adapter.js    293 KiB       7  [emitted]        [big]  vendors~enzyme-adapter
       vendors~enzyme-adapter.js.map    995 KiB       7  [emitted] [dev]         vendors~enzyme-adapter
    vendors~enzyme~enzyme-adapter.js    534 KiB       0  [emitted]        [big]  vendors~enzyme~enzyme-adapter
vendors~enzyme~enzyme-adapter.js.map   2.21 MiB       0  [emitted] [dev]         vendors~enzyme~enzyme-adapter
Entrypoint frame-runner = frame-runner.js frame-runner.js.map
Entrypoint sass-compile = sass-compile.js sass-compile.js.map
Entrypoint test-evaluator = test-evaluator.js test-evaluator.js.map
[322] (webpack)/buildin/global.js 472 bytes {0} {5} {6} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
[365] ./src/client/frame-runner.js 5.13 KiB {3} [built]
      ModuleConcatenation bailout: Module uses eval()
[369] ./src/client/workers/sass-compile.js 756 bytes {4} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
[370] ./src/client/workers/test-evaluator.js 3.7 KiB {5} [built]
      ModuleConcatenation bailout: Module uses eval()
[399] (webpack)/buildin/module.js 497 bytes {0} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
[494] readable-stream (ignored) 15 bytes {1} {2} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
[503] util (ignored) 15 bytes {1} {2} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
[504] util (ignored) 15 bytes {1} {2} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
[567] ./util.inspect (ignored) 15 bytes {1} {2} [built]
      ModuleConcatenation bailout: Module is not an ECMAScript module
    + 847 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  vendors~enzyme~enzyme-adapter.js (534 KiB)
  vendors~enzyme-adapter.js (293 KiB)
  sass.sync.js (4.27 MiB)

> @freecodecamp/curriculum@0.0.0-next.4 test D:\Coding\fcc\curriculum
> mocha --delay --reporter progress --bail

Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
| Populate tests.
No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In D:\Coding\fcc
  552 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 39 matches
  testPathIgnorePatterns: \\node_modules\\ - 552 matches
  testRegex:  - 0 matches
Pattern: .\\utils\\*.js - 0 matches
- Testingnpm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @freecodecamp/freecodecamp@0.0.1 test:utils: `jest ./utils/*.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @freecodecamp/freecodecamp@0.0.1 test:utils script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\OPT\AppData\Roaming\npm-cache\_logs\2019-10-29T13_32_32_758Z-debug.log
| TestingERROR: "test:utils" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @freecodecamp/freecodecamp@0.0.1 test: `npm-run-all -p test:*`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @freecodecamp/freecodecamp@0.0.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\OPT\AppData\Roaming\npm-cache\_logs\2019-10-29T13_32_33_055Z-debug.log
```

**Comment by RandellDawson** [comment]
@raisedadead I raised this issue in the Contributors room, but only @moT01 responded that he was not seeing the same issue.

**Comment by ojeytonwilliams** [comment]
@RandellDawson pretty sure it's a Windows path issue, so Tom wouldn't have run into it.

If you could try replacing the `test:utils` npm script with the following, we can see if that fixes it.
```json
"test:utils": "bash -c 'jest ./utils/*.js'"
```

**Comment by RandellDawson** [comment]
@ojeytonwilliams, @raisedadead Replacing the `test:utils` with your suggestion now allows the tests to run without the error.

**Comment by Djisu** [comment]
No, when I enter npm ci, it hangs, telling me the following in the attached file.
I realized that npm install or npm ci does not work well in windows 10. I tried to delete the node_modules folder from my root folder and issued npm install node to solve the problem. So that if it node is successfully installed, I will re-try 'npm ci' again. The problem persists hanging at:
[.........] X installing dependencies: verb actions 1 actions, concurrency 4. I will be glad if provide a solution. Thanks.
![npm ci hanging page](https://user-images.githubusercontent.com/22422354/67981038-fe56f780-fbdc-11e9-8c74-237e6515b3d5.png).
